test/dlc_tests/sanity/test_temp_config_reverted.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+from src.config.test_config import ENABLE_BENCHMARK_DEV_MODE
+
+
+@pytest.mark.model("N/A")
```

Comment:
In case the config change is not reverted before a PR been merged, this test would catch this error on mainline/nightly pipeline. Here is a sample error message, extracted from codebuild log
```
@pytest.mark.model("N/A")
  def test_benchmark_dev_mode_disabled():
  >       assert not ENABLE_BENCHMARK_DEV_MODE, \
  "Please revert the config change on src/config/test_config.py by setting ENABLE_BENCHMARK_DEV_MODE = False"
  E       AssertionError: Please revert the config change on src/config/test_config.py by setting ENABLE_BENCHMARK_DEV_MODE = False
  E       assert not True
```